assets/settings/initial_debug_tasks.json

```diff
@@ -1,28 +1,28 @@
 [
   {
     "label": "Debug active PHP file",
-    "adapter": "php",
+    "adapter": "PHP",
     "program": "$ZED_FILE",
     "request": "launch",
     "cwd": "$ZED_WORKTREE_ROOT"
   },
   {
     "label": "Debug active Python file",
-    "adapter": "python",
+    "adapter": "Debugpy",
     "program": "$ZED_FILE",
     "request": "launch",
     "cwd": "$ZED_WORKTREE_ROOT"
   },
   {
     "label": "Debug active JavaScript file",
-    "adapter": "javascript",
+    "adapter": "JavaScript",
     "program": "$ZED_FILE",
     "request": "launch",
     "cwd": "$ZED_WORKTREE_ROOT"
   },
   {
     "label": "JavaScript debug terminal",
-    "adapter": "javascript",
+    "adapter": "JavaScript",
     "request": "launch",
     "cwd": "$ZED_WORKTREE_ROOT",
     "initialize_args": {
```

Comment:
We should also inline these initialize_args here ig

Reply:
Oh, good call